spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/EventProcessorProperties.java

```diff
@@ -32,10 +32,12 @@
 
```

Comment:
A test case to ensure the settings are propagated as expected is a hard requirement.

Reply:
I do agree. Done.

Reply:
I do agree. Done.